modules/renter/hostdb/scan.go

```diff
@@ -272,11 +287,6 @@ func (hdb *HostDB) managedScanHost(entry modules.HostDBEntry) {
 
 // threadedProbeHosts pulls hosts from the thread pool and runs a scan on them.
 func (hdb *HostDB) threadedProbeHosts(scanPool <-chan modules.HostDBEntry) {
-	err := hdb.tg.Add()
-	if err != nil {
-		return
-	}
-	defer hdb.tg.Done()
```

Comment:
Is there a reason that we are eliminating this call to `tg`? I don't think we want the hostdb returning successfully after a call to close unless `threadedProbeHosts` has shut down, which we achieve by adding it to the `tg`.